packages/core/package.json

```diff
@@ -10,6 +10,7 @@
     "test": "jest",
     "test::watch": "jest --watch",
     "test::unit": "npm run test",
+    "test::deno": "deno test --allow-none ./test/deno/",
```

Comment:
It might worth to remove the flag `--allow-none` after you create testing code under `./test/deno/`.